utils/ccdatofhir/mapping_resource_procedure.bal

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// --------------------------------------------------------------------------------------------#
+// Source C-CDA to FHIR - Resource Mappings
+// --------------------------------------------------------------------------------------------#
+
+import ballerina/log;
+import ballerinax/health.fhir.r4;
+
+# Map CCDA Procedure Activity to FHIR Procedure
+#
+# + actElement - CCDA Procedure Activity Element
+# + isNamespaceAvailable - Is CCDA namespace available
+# + return - FHIR Procedure Resource
+public isolated function mapCcdaProcedureToFhir(xml<xml:Element> actElement, boolean isNamespaceAvailable = true) returns r4:Procedure?|error {
```

Comment:
Same problem here.
```suggestion
public isolated function mapCcdaProcedureToFhir(xml<xml:Element> actElement, boolean isNamespaceAvailable = true) returns r4:Procedure? {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/817dadb5178bde7d809333a26f465c6688d3879a